packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -241,6 +249,13 @@ class Auth {
 
         this.redirectURI = config.redirectURI
 
+        this.passwordlessConfig = {
```

Comment:
From the logic we have here the callbackURI value is optional as it will always be populated with at least the baseUrl? 
Is that really desired or a real world scenario? Should we be warning the user if they provide a passwordlessConfig, but don't provide a callbackURI within it? It might help them diagnose tricky issues if one isn't define properly.

Reply:
I might be looking a little bit too much into this, but it might be weird that we always have a `passwordlessConfig` on the objects scope because it might not always exist in the configuration. This might lead to other parts of your code behaving unexpectedly.